src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -312,24 +327,43 @@ def _forward(self, model_inputs):
             out = {"tokens": tokens}
         else:
             stride = model_inputs.pop("stride", None)
-            input_values = model_inputs.pop("input_values")
+            input_values = (
+                model_inputs.pop("input_values")
+                if "input_values" in model_inputs
+                else model_inputs.pop("input_features")
+            )
             attention_mask = model_inputs.pop("attention_mask", None)
-            outputs = self.model(input_values=input_values, attention_mask=attention_mask)
+            outputs = self.model(input_values, attention_mask=attention_mask)
```

Comment:
M-CTC-T takes an argument `input_features` instead of `input_values`, so I made this a positional argument instead of a keyword argument. I'm happy to add some if/else logic to keep it a keyword argument if desired, but this is a bit easier to read.